openrpc/src/stellar-rpc/methods/simulateTransaction.json

```diff
@@ -85,8 +85,12 @@
                         "type": "object",
                         "properties": {
                             "type": {
-                                "type": "integer",
-                                "enum": [1, 2, 3],
+                                "type": "string",
+                                "enum": [
+                                    "created",
+                                    "updated",
+                                    "deleted"
+                                ],
                                 "description": "Indicates if the entry was created (1), updated (2), or deleted (3)"
```

Comment:
You might wanna drop the integers from this, though!

Reply:
good catch!